src/Model/GroundWaterFlow/gwf3uzf8.f90

```diff
@@ -23,8 +23,9 @@ module UzfModule
   use ObserveModule, only: ObserveType
   use ObsModule, only: ObsType
   use InputOutputModule, only: URWORD
-  use SimVariablesModule, only: errmsg
-  use SimModule, only: count_errors, store_error, ustop, store_error_unit
+  use SimVariablesModule, only: errmsg, warnmsg
+  use SimModule, only: count_errors, store_error, ustop, store_error_unit,      &
+                       store_warning
```

Comment:
Probably don't need store_warning any more

Reply:
or warnmsg

Reply:
Have removed the adjustments that included warnmsg